packages/nextjs/package.json

```diff
@@ -26,22 +26,34 @@
   },
   "devDependencies": {
     "@sentry/types": "6.2.5",
+    "@sentry/utils": "6.2.5",
+    "@types/prettier": "^2.2.3",
+    "chokidar-cli": "^2.1.0",
     "eslint": "7.20.0",
     "rimraf": "3.0.2"
   },
   "scripts": {
-    "build": "run-p build:esm",
-    "build:esm": "tsc -p tsconfig.esm.json",
+    "build": "run-s build:esm",
+    "build:scripts": "tsc -p tsconfig.scripts.json",
+    "build:types": "run-s build:scripts generate:types",
```

Comment:
TBH i'd not build it like this, as it adds unnecessary overhead. If we _want_ types in build script, let's use `ts-node` instead.